lottie/src/main/res/values/attrs.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <declare-styleable name="LottieAnimationView">
-        <attr name="lottie_fileName" format="string" />
+        <attr name="lottie_setAnimation" format="string" />
```

Comment:
We definitely can't break the existing API here. Hundreds of people use it. It would have to at least fall back on the new one.
I would consider `lottie_asset` to be consistent with `rawRes` though. If we wanted to make them more parallel, we could deprecate `setAnimation` in favor of `setAsset` as well.

Reply:
@gpeal 
I agree with you 👍 
Can I make a new PR that following features?
- add deprecate annotation to `setAnimation()`
- add new method `setAsset()` or etc